deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpListener.java

```diff
@@ -8,11 +8,5 @@
 {
     void messageReceived(AmqpMessage message);
 
-    void connectionEstablished();
-
-    void connectionLost();
-
-    void messageSent();
```

Comment:
The implementations of this interface did nothing when these functions were called